docs/pages/preview/upcoming-releases.mdx

```diff
@@ -22,13 +22,13 @@ You can find the full list of fixes and features in the [GitHub milestone](https
 
 ### Teleport 9
 
-Teleport 9.0 will focus on bringing Desktop Access to GA with support for
-session recording, per-session MFA, and clipboard support. Database Access
-will add self-hosted Redis, auto-discovery for Redshift clusers, and
-auto-IAM configuration improvements.
+Teleport 9.0 will focus on bringing the Desktop Service to GA with support for
```

Comment:
This change probably isn't necessary. Nobody should be going to the upcoming releases page anymore to find out whats in a release from 9 months ago.